orderedmap.go

```diff
@@ -4,7 +4,6 @@
 // All operations are constant-time.
 //
 // Github repo: https://github.com/wk8/go-ordered-map
-//
```

Comment:
I don't get that diff when I run go fmt, and the linter also runs go fmt anyhow, so not sure how you get this?

Reply:
Hm interesting. I am running `go version go1.19.4 linux/amd64` as the fmt call in the title.

Feel free to close this if you'd like. No strong opinions here.

Reply:
Thanks for your understanding, closing